### PR TITLE
Created tests for assignments_controller

### PR DESCRIPTION
**Pull request by GabrielRMuller**
related to #862

### CODE REVIEW

**Comment by jhsu802701** [comment]
This build failed because of RuboCop offenses and not because of test failures.  Please fix these RuboCop offenses.

**Comment by GabrielRMuller** [comment]
oops! forgot to check these things, the fixes were made, sorry about that!

File: test/functional/issue_assignments_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+require 'test_helper'
+
+class IssueAssignmentsControllerTest < ActionController::TestCase
+  include Devise::Test::ControllerHelpers
+
+  fixtures :users, :repo_subscriptions, :issue_assignments
+
+  test '#create redirects when the user request a new issue' do
+    sign_in users(:schneems)
```
Comment (prathamesh-sonpatki) [inline test/functional/issue_assignments_controller_test.rb]: Let's move this to the setup block as it is common to all the tests.

File: test/functional/issue_assignments_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+require 'test_helper'
+
+class IssueAssignmentsControllerTest < ActionController::TestCase
+  include Devise::Test::ControllerHelpers
+
+  fixtures :users, :repo_subscriptions, :issue_assignments
```
Comment (prathamesh-sonpatki) [inline test/functional/issue_assignments_controller_test.rb]: We don't need to load individual fixtures as all fixtures are loaded by https://github.com/codetriage/codetriage/blob/af1de44c5075546e9b5d6ce2cff166fe678bbe99/test/test_helper.rb#L19

Comment (GabrielRMuller) [reply test/functional/issue_assignments_controller_test.rb]: done!

**Pull request merged**